src/grpc/hpr_gateway_service.erl

```diff
@@ -0,0 +1,37 @@
+-module(hpr_gateway_service).
+
+-behaviour(helium_packet_router_gateway_bhvr).
+
+-include("../grpc/autogen/server/packet_router_pb.hrl").
+
+-export([
+    init/2,
+    send_packet/2,
+    handle_info/2
+]).
+
+-spec init(atom(), StreamState :: grpcbox_stream:t()) -> grpcbox_stream:t().
+init(_RPC, StreamState) ->
+    StreamState.
+
+-spec send_packet(hpr_packet_up:packet(), grpcbox_stream:t()) ->
+    {ok, grpcbox_stream:t()} | grpcbox_stream:grpc_error_response().
+send_packet(PacketUp, StreamState) ->
+    grpcbox_service_reply(hpr_routing:handle_packet(PacketUp), StreamState).
```

Comment:
We want to keep the spawn to `hpr_routing` here. 
If there's no destination for the packet, we just don't say anything back to the gateway.

Reply:
Mack says the spawn is not needed. There is already a process per request. How does another spawn here help any?

Reply:
You're correct. I was conflating the 2 grpc flows. :+1: